EIPS/eip-3779.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 3779
+title: Safe Control Flow for the EVM
```

Comment:
Description field missing.  Filled in a placeholder in the suggestion below, feel free to put whatever you want though.  😄 
```suggestion
title: Safe Control Flow for the EVM
description: Introduces contract validation rules for jumps and stack depth that are applied at deploy time.
```